src/server/enterprise/server/enterprise_test.go

```diff
@@ -302,3 +302,136 @@ func TestEnterpriseConfigMigration(t *testing.T) {
 	require.YesError(t, err)
 	require.True(t, collection.IsErrNotFound(err))
 }
+
+/*
+   N.b.: for these tests to run successfully on Linux I needed to upgrade to the
+   latest kubectl and run port forwards in a _loop_.  I.e.:
+
+     while :; do  kubectl port-forward svc/pachd 30650:1650; done
+*/
+func TestPauseUnpause(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	testutil.DeleteAll(t)
+	defer testutil.DeleteAll(t)
+	client := testutil.GetPachClient(t)
+
+	// Activate Pachyderm Enterprise and make sure the state is ACTIVE
+	testutil.ActivateEnterprise(t, client)
+	testutil.ActivateAuth(t)
+
+	_, err := client.Enterprise.Pause(client.Ctx(), &enterprise.PauseRequest{})
+	require.NoError(t, err)
+	bo := backoff.NewExponentialBackOff()
+	backoff.Retry(func() error {
+		resp, err := client.Enterprise.PauseStatus(client.Ctx(), &enterprise.PauseStatusRequest{})
+		if err != nil {
+			return errors.Errorf("could not get pause status %w", err)
+		}
+		if resp.Status == enterprise.PauseStatusResponse_PAUSED {
+			return nil
+		}
+		return errors.Errorf("status: %v", resp.Status)
+	}, bo)
+
+	// ListRepo should return an error since the cluster is paused now
+	_, err = client.ListRepo()
+	require.NotNil(t, err)
```

Comment:
nit: for consistency, we should use require.NoError() here

Reply:
In this case, an error is expected (thus the `NotNil`), so I think it would be `require.YesError`. Updating.